# Goal
geometry_msgs/PoseStamped target_pose
---
# Result
---
# Feedback
string stat
geometry_msgs/Pose position

